google/cloud/forseti/services/scanner/dao.py

```diff
@@ -120,8 +127,10 @@ def create(self, violations, inventory_index_id):
                         violation_type=violation.get('violation_type'),
                         violation_data=json.dumps(
                             violation.get('violation_data')),
-                        inventory_data=violation.get('inventory_data')
+                        inventory_data=violation.get('inventory_data'),
+                        violation_hash=violation_hash
                     )
+
                     session.add(violation)
```

Comment:
This might to handle if violation can not be added due to hash is not unique.

Reply:
Done.